docs/en/advanced-features/kafka-reporter.md

```diff
@@ -0,0 +1,147 @@
+# Kafka Reporter
+
+This document describes how to configure and use the Kafka reporter in the Apache SkyWalking Go agent. The Kafka reporter provides an alternative to the default gRPC reporter, allowing you to send trace, metrics, and log data to Apache Kafka.
+
+## Overview
+
+The SkyWalking Go agent can be configured to report collected telemetry data (traces, metrics, logs) to a Kafka cluster. This is useful for scenarios where Kafka is already part of your infrastructure or when you prefer Kafka's buffering and scalability features for handling observability data.
```

Comment:
We should give a clear note, the gRPC connection(OAP backend/port) is still required for command-pushing and other features.